### DISCUSSION TITLE
Help trying to get the example up and running.

### DISCUSSION THREAD

**Discussion by rolfwessels** [eclipse-tractusx/tractusx-edc#676] [Q&A]
Hi,

I've been trying to get https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/samples/example-dataspace/README.md up and running but it is not clear what to do.

The document says "Please refer to the respective manuals and guide" to setup MIW but this https://github.com/catenax-ng/tx-managed-identity-wallets#deployment-on-kubernetes does really define how to setup the environment or what each of the <placeholders> represents. I could not find any other documentation or working example.

Can you please point me in the right direction?

Kind regards

Rolf

**Comment by stephanbcbauer** [comment]
@rolfwessels thanks for starting the discussion. Is this a redundant discussion to #696  ?

**Reply by rolfwessels** [reply]
Nope, @HenkySerpent seems to have gotten further than me, I did not get KeyCloak up and running with MIW. This is a pre-requisite for sokrates and plato

**Reply by HenkySerpent** [reply]
Hi, I used this repo : [tx-miw](https://github.com/catenax-ng/tx-managed-identity-wallets/tree/b420443dff0a6ac5a7fcdb19f013231a416676d8) for MIW.

**Reply by rolfwessels** [reply]
I tried that but the when trying to follow this > https://github.com/catenax-ng/tx-managed-identity-wallets/tree/b420443dff0a6ac5a7fcdb19f013231a416676d8#deployment-on-kubernetes I was unclear of what the <placeholders>values should be,

**Reply by HenkySerpent** [reply]
I used local setup and just followed the instructions they provided

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
